neptune/new/attribute_container.py

```diff
@@ -135,6 +135,11 @@ def __getattr__(self, item):
     def _label(self) -> str:
         raise NotImplementedError
 
+    @property
+    @abc.abstractmethod
+    def _docs_url(self) -> str:
```

Comment:
Should this be `_docs_url` or maybe sth like `_docs_url_stop` as it always returns a link to stop method. When adding new class that inherit from attribute container it would be easier probably to understand what need to be done?

Reply:
Ok